indexing-service/src/main/java/org/apache/druid/indexing/common/stats/TaskRealtimeMetricsMonitor.java

```diff
@@ -50,6 +50,8 @@ public class TaskRealtimeMetricsMonitor extends AbstractMonitor
   private FireDepartmentMetrics previousFireDepartmentMetrics;
   private RowIngestionMetersTotals previousRowIngestionMetersTotals;
 
+  private volatile boolean lastRoundMetricsToBePushed = true;
```

Comment:
Wouldn't it be better to initialize this to `false` and set it to true in an overriding `start` method for this class?
1) It would prevent emission of metrics before the monitor has actually started
2) It may allow for reuse (Might not be intended)

Reply:
I'm thinking if this can be made `return doMonitor(emitter) && started;` removing the necessity for `lastRoundMetricsToBePushed`